salt/modules/mount.py

```diff
@@ -1262,3 +1263,92 @@ def is_mounted(name):
         return True
     else:
         return False
+
+
+def read_mount_cache(name):
+    '''
+    .. versionadded:: Oxygen
+
+    Provide information if the path is mounted
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' mount.read_mount_cache /mnt/share
+    '''
+    cache = salt.utils.mount.read_cache(__opts__)
+    if cache:
+        if 'mounts' in cache and cache['mounts']:
+            if name in cache['mounts']:
+                return cache['mounts'][name]
+    return {}
+
+
+def write_mount_cache(real_name,
+                      device,
+                      mkmnt,
+                      fstype,
+                      opts):
+    '''
+    .. versionadded:: Oxygen
+
+    Provide information if the path is mounted
+
```

Comment:
Can you add definitions to the docs here for what passed in args represent? You're using the `opts` option, which I think can be confused with regular `__opts__` and I'd like to be more clear about this up front with the docs.